stylesheets/commons/Miscellaneous.css

```diff
@@ -4791,19 +4791,25 @@ Author(s): hjpalpha
 
 .Zerg,
 .warcraft-orc,
+.stormgate-infernal,
```

Comment:
could do something like
```suggestion
.participantTable-faction-header.stormgate-infernal,
```
to only add them for participant tables for now

Reply:
don't have an easy option to show race icons in brackets yet
until then i would like to show them in brackets too
it is on my to do to change that in the sc(2) match stuff early next year
but other stuff has prio for now

Reply:
as per the poll in the channel race bg in brackets is wanted (even though race icon is now enabled there too)